portal-api/src/v1/gef/models/facilities.js

```diff
@@ -81,15 +82,17 @@ class Facility {
       }
 
       if (req.coverStartDate != null) {
-        this.coverStartDate = new Date(req.coverStartDate);
+        const timestamp = convertToTimestamp(req.coverStartDate);
+        this.coverStartDate = new Date(timestamp);
       }
 
       if (req.coverEndDate != null) {
-        this.coverEndDate = new Date(req.coverEndDate);
+        const timestamp = convertToTimestamp(req.coverEndDate);
+        this.coverEndDate = new Date(timestamp);
       }
 
       if (req.issueDate != null) {
-        this.issueDate = new Date(req.issueDate);
+        this.issueDate = Date(req.issueDate);
```

Comment:
Does this not require creating a new date class object - `new Date()` ?

Reply:
fixed